ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/configs/DynamoDBConfigs.java

```diff
@@ -0,0 +1,42 @@
+package com.netflix.ndbench.plugin.dynamodb.configs;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+/**
+ * Configurations for DynamoDB benchmarks
+ *
+ * @author ipapapa
+ */
+@Configuration(prefix = "ndbench.config.dynamodb")
+public interface DynamoDBConfigs {
```

Comment:
This looks interesting. I am curious about bindings though. Hopefully, this does not require explicit bindings.

Reply:
Good catch, I missed that. I need to add a `NdBenchClientPluginGuiceModule` to leverage the custom configuration. I will do it.

Reply:
Fixed it!